bigtable/admin.go

```diff
@@ -275,6 +275,67 @@ func (ac *AdminClient) CreateColumnFamily(ctx context.Context, table, family str
 	return err
 }
 
+// set to true to make the table protected against data loss
```

Comment:
Type comments should start with the name of the type. 

Suggestion: "DeletionProtection indicates whether the table is protected ..."